bin/reth/src/args/types.rs

```diff
@@ -58,6 +58,31 @@ impl FromStr for MaxU32 {
     }
 }
 
+macro_rules! max_values {
+    ($name:ident, $ty:ident) => {
+        #[derive(Debug, Clone, Copy, PartialEq, Eq)]
```

Comment:
needs one line doc

Reply:
> needs one line doc

wdym ? there is already docs, what should i add?